src/web-ui/src/router/index.js

```diff
@@ -72,7 +72,7 @@ AmplifyEventBus.$on('authState', async (state) => {
       storeUser = data
     }
     else {
-      const { data } = await UsersRepository.getUserByUsername(cognitoUser.username)
+      const { data } = await UsersRepository.getUserByUsername(cognitoUser.attributes.email)
```

Comment:
Since username is going away, the Users service endpoints should change as well from `getUserByUsername` to `getUserByEmail`.